config/rubocop.yml

```diff
@@ -7,10 +7,11 @@ AllCops:
     - bin/**
     - vendor/bundle/**/*
     - node_modules/**/*
+  NewCops: disable
```

Comment:
why?

Reply:
Because new crops can affect style of our code in way that we don't want, and due to that I wasn't sure if it will be right approach.
I can enable all of new cops, and check how they will affect our code. Or just, enable it. Please let me know what you think.